### PR TITLE
Deploy automatically an example app on PRs

### PR DESCRIPTION
**Pull request by xavier-villelegier**
Refs #1022

### CODE REVIEW

**Comment by iRoachie** [comment]
Example app for 98de54a2277a9d1150e1b154055d4bf785531b04:

QR Code

https://exp.host/@xavier-villelegier/react-native-elements-app?release-channel=98de54a2277a9d1150e1b154055d4bf785531b04

**Comment by xavier-villelegier** [comment]
>  xavier-villelegier deleted a comment from iRoachie 11 minutes ago

cc @iRoachie There was still your token Travis, my bad 😆

**Comment by xavier-villelegier** [comment]
@Monte9 @iRoachie Okay nice that works fine 👆
(At least we can see that the buttons are broken on the `next` branch lol. We need to review #1000 )

File: .travis.yml [hunk 1]
```diff
@@ -17,11 +17,10 @@ install:
 - yarn
 script:
 - yarn run test && codecov
+after_script:
+- ./.travis/expo-ci.sh
```
Comment (iRoachie) [inline .travis.yml]: might wanna change here too :P

Comment (xavier-villelegier) [reply .travis.yml]: Oh, this one is rude

**Comment by iRoachie** [comment]
@xavier-villelegier is there anyway for the bot to update the comment instead of creating a new comment each time? This could get pretty noisy on long PRs

**Comment by xavier-villelegier** [comment]
@iRoachie Yea had added that to my backlog, i've been thinking about that as an enhancement. This implies reading the comments first to check if there is already one, which becomes a bit tricky in shell script. I'll take some time later to improve that with a node script

**Comment by iRoachie** [comment]
Okay great. The logic around it could be checking the author username in the comments

**Comment by xavier-villelegier** [comment]
@iRoachie Yes sure, in js this can be done pretty easily (cf [Github PR comments API](https://developer.github.com/v3/pulls/comments/#response-1))

**Comment by iRoachie** [comment]
Would it be easier to write a node script then?

**Comment by xavier-villelegier** [comment]
@iRoachie Yeaa, that's basic API calls, really not a big deal. I'll do it next week end

**Pull request merged**